guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -245,7 +247,7 @@ namespace.Storage.prototype.getLastId&nbsp;=&nbsp;function()&nbsp;{
 };
 </pre>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-this">@this</h4>
+#### @this
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-this}` to preserve the custom id.